c-example/Makefile

```diff
@@ -28,3 +34,6 @@ example_static$(EXE):  example.c ../test_models/$(MODEL)/$(MODEL)_model.a
 	$(CC) -c -I ../src example.c -o example.o
 	$(LINK.cpp) -o example_static$(EXE) example.o ../test_models/$(MODEL)/$(MODEL)_model.a $(LDLIBS) $(LIBSUNDIALS) $(MPI_TARGETS) $(TBB_TARGETS)
 	$(RM) example.o
+
+example_runtime$(EXE): runtime_loading.c
```

Comment:
Is it reasonable to add `../test_models/$(MODEL)/lib$(MODEL)_model.$(DLL)` here?

Reply:
It would cause `example_runtime` to be re-built more often, which feels like it goes against the pedagogical purpose of the example. Strictly speaking, there is no reason that any bridgestan library needs to have been compiled before this, which is what that dependency would be saying

Reply:
> goes against the pedagogical purpose of the example

Right, good point. 

Is it reasonable to leave the code suggestion in the Readme as is, then add a sentence or two explaining that `example_runtime` the executable can be compiled, but not run without a `.so`?

Reply:
I updated the readme block:

>
```shell
make example_runtime
# unlike above, this did not automatically build a model, since it
# was not needed to _build_, but we still need one to _run_ the program
make ../test_models/full/full_model.so
./example_runtime ../test_models/full/full_model.so
```